api/src/main/java/ai/djl/nn/AbstractBlock.java

```diff
@@ -122,7 +124,7 @@ public int getVersion() {
      * @return the block given as a parameter - that way the block can be created and reassigned to
      *     a member variable more easily.
      */
-    protected <B extends Block> B addChildBlock(String name, B block) {
+    protected final <B extends Block> B addChildBlock(String name, B block) {
```

Comment:
These method is called in constructor, they must be final

Reply:
@frankfliu Do you want me to change that?

Reply:
@chenkelmann I already added final to those methods. I just add comment here to show why this final is necessary.